### PR TITLE
simplified env file

### PR DESCRIPTION
**Pull request by TimCsaky**
# Description

added the auth_host and cdogs_host environemnt variables to the example template.
removed some unnecessary properties generated by postman

## Types of changes

Documentation (non-breaking change with enhancements to documentation)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have updated the OpenAPI 3.0 `v*.api-spec.yaml` documentation (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/tests/postman/example.postman_environment.json [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
+			"key": "auth_host",
+			"value": "https://sso-dev.pathfinder.gov.bc.ca",
+			"enabled": true
+		},
+		{
+			"key": "cdogs_host",
+			"value": "https://cdogs-master-idcqvl-dev.pathfinder.gov.bc.ca",
+			"enabled": true
+		},
+
```
Comment (jujaga) [inline app/tests/postman/example.postman_environment.json:13]: unnecessary whitespace line here.

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-idcqvl-tools.pathfinder.gov.bc.ca/dashboard?id=common-document-generation-service-pr-41
